_docs/layout/grid.md

```diff
@@ -2,7 +2,7 @@
 title: Grid
 layout: docs
 category: Layout
-usage: The Helix 12-column grid provides structure for website content.
+usage: The Helix 12 Column Grid provides structure for website content.
```

Comment:
I don't think this should be capitalized. The grid is a generic thing, not something specific to Helix. And "12-column" is a descriptor as that's a type of grid..